File: LLD-C#/MessageQueue/Models/QueueMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MessageBroker.Models
{
    internal class QueueMessage
    {
        public required string Id { get; set; }
        public required string Message { get; set; }
        public Object? AdditionalProperties { get; set; }
    }
}

File: LLD-Java/MessageQueue/src/Models/QueueMessage.java
package Models;

public class QueueMessage {
    private final String id;
    private final String message;
    private final Long createdAt;

    public QueueMessage(String id, String message, Long createdAt){
        this.id=id;
        this.message=message;
        this.createdAt = createdAt;
    }

    public String getMessage(){
        return message;
    }

    public String getId(){
        return id;
    }

    public Long getCreatedAt(){
        return createdAt;
    }
}
